test/spec/memory.wast

```diff
@@ -181,54 +90,6 @@
     )
   )
 
-  ;; Aligned read/write
-  (func (export "aligned") (result i32)
-    (local i32 i32 i32)
-    (local.set 0 (i32.const 10))
-    (block
-      (loop
-        (if
-          (i32.eq (local.get 0) (i32.const 0))
-          (br 2)
-        )
-        (local.set 2 (i32.mul (local.get 0) (i32.const 4)))
-        (i32.store (local.get 2) (local.get 0))
-        (local.set 1 (i32.load (local.get 2)))
-        (if
-          (i32.ne (local.get 0) (local.get 1))
-          (return (i32.const 0))
-        )
-        (local.set 0 (i32.sub (local.get 0) (i32.const 1)))
-        (br 0)
-      )
-    )
-    (i32.const 1)
-  )
-
-  ;; Unaligned read/write
-  (func (export "unaligned") (result i32)
-    (local i32 f64 f64)
-    (local.set 0 (i32.const 10))
-    (block
-      (loop
-        (if
-          (i32.eq (local.get 0) (i32.const 0))
-          (br 2)
-        )
-        (local.set 2 (f64.convert_i32_s (local.get 0)))
-        (f64.store align=1 (local.get 0) (local.get 2))
-        (local.set 1 (f64.load align=1 (local.get 0)))
-        (if
-          (f64.ne (local.get 2) (local.get 1))
-          (return (i32.const 0))
-        )
-        (local.set 0 (i32.sub (local.get 0) (i32.const 1)))
-        (br 0)
-      )
-    )
-    (i32.const 1)
-  )
```

Comment:
Why are these tests being deleted? Were they suitably replaced elsewhere?

Reply:
I'm not sure - this is just an update from the upstream spec suite.

Reply:
These have been moved to and expanded in align.wast a while ago, see (here)[https://github.com/WebAssembly/spec/pull/822]. Various other tests have since been moved to separate files as well, esp. load.wast and store.wast.

Reply:
Thanks, @rossberg!